.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,14 @@
+name: Python package
+on:
+  release:
+    types: [published]
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build and publish to pypi
+        uses: JRubics/poetry-publish@v1.17
```

Comment:
what made you choose this action specifically instead of other poetry github actions? like: 
https://github.com/marketplace/actions/python-poetry-action
or
https://github.com/marketplace/actions/install-poetry-action

Reply:
I think, `JRubics/poetry-publish@v1.17`, used for building and publishing python packages to pypi, pypi-test or a private wheels repo using poetry. and the two you mentioned here mostly focus on actions installing and configuring poetry.